### ISSUE TITLE
Data information

### ISSUE THREAD

**Issue by RuffaloLavoisier** [Open-Smartwatch/open-smartwatch-os#207]
I would like to provide information to Watchface. 

TO-DO:
---
* [x] Battery Notification (Arc or anything)
* [ ] Provides sunrise
* [x] Extend fitness app : https://github.com/Open-Smartwatch/open-smartwatch-os/pull/223
  * [x] Total distance
  * [x] Total calories consumed  
  * [x] Total steps
* [ ] Draw tracking map (With GPS)
>Bluetooth Applications to be developed in the future
---
I think we can provide this kind of information at the current step counter.

I think it is possible to extract sufficiently valuable data by storing user information(height, weight) that needs to be received by the Open-smartwatch in config.h or OSW config.

---
Related issue :

* https://github.com/Open-Smartwatch/open-smartwatch-os/issues/23

**Comment by RuffaloLavoisier** [comment]
See https://github.com/Open-Smartwatch/open-smartwatch-os/pull/223

**Comment by RuffaloLavoisier** [comment]
It's time to restart this part.

**Comment by simonmicro** [comment]
@Lorenzosciacca I think this is a GREAT idea and - from the looks - beautiful implementation! I think using openweather is OK, as e.g. https://github.com/sqfmi/Watchy/blob/5690db077f5c99c7823cb652009bc260605b231f/examples/WatchFaces/7_SEG/settings.h#L6 is also doing that. I'm very impressed!

**Comment by RuffaloLavoisier** [comment]
@Lorenzosciacca If you want, could you implement this weather part?

**Comment by Lorenzosciacca** [comment]
@simonmicro  thank you!
@RuffaloLavoisier , of course, it will be a pleasure for me.

**Comment by RuffaloLavoisier** [comment]
@Lorenzosciacca 
Thank you in advance ☺

**Comment by Lorenzosciacca** [comment]
flash update: this is how the app looks like now. While i was playing with the smartwatch the screen died, it’s the usual problem but this slowed down everything. In the next future, when I will be able to do a couple of tests, I will post everything.

https://user-images.githubusercontent.com/60114811/189479755-361d1fea-7265-46be-91ac-520c34bfa87e.mp4

**Comment by mat942** [comment]
Looks great, love the UI!

**Comment by Lorenzosciacca** [comment]
The app is available [here](https://github.com/Lorenzosciacca/OSW_weather).
Note: there is a bug, when i try to perform the GET to openweather i can't enable the wifi directly from the app, instead it's necessary to enable the WiFi from the webserver/time-update app. I don't know the exact reason, it also may be related to the usual WiFi problems.

**Comment by RuffaloLavoisier** [comment]
@Lorenzosciacca Could you open the pull request?

**Comment by Lorenzosciacca** [comment]
Yes, i can, now i'm not at home, tomorrow morning i can do It for sure. On develop right?

**Comment by RuffaloLavoisier** [comment]
Yes, correct
